Description: buildflags only set -buildmode=pie on linux, which we control with
GOFLAGS. -buildmode=pie is not supported on mips.
diff --git a/build.sh b/build.sh
index 519f9c8..c96ed9c 100755
--- a/build.sh
+++ b/build.sh
@@ -29,7 +29,7 @@ mkdir -p bin
 ec=0
 for i in bsondump mongostat mongofiles mongoexport mongoimport mongorestore mongodump mongotop mongoreplay; do
         echo "Building ${i}..."
-        go build -o "bin/$i$BINARY_EXT" $(buildflags) -ldflags "$(print_ldflags)" -tags "$(print_tags $tags)" "$i/main/$i.go" || { echo "Error building $i"; ec=1; break; }
+        go build -o "bin/$i$BINARY_EXT" -ldflags "$(print_ldflags)" -tags "$(print_tags $tags)" "$i/main/$i.go" || { echo "Error building $i"; ec=1; break; }
         ./bin/${i}${BINARY_EXT} --version | head -1
 done
 
